docs/swagger.yaml

```
@@ -773,4 +782,25 @@ paths:
+      responses:
+        "200":
+          description: OK
+        "400":
+          description: Bad Request
+        "404":
+          description: Not Found
+        "500":
+          description: Internal Server Error
+      summary: changes owner to new owner with uuid
+      tags:
+      - vehicle
```

YAMLlint (1.35.1)

[error] wrong indentation: expected 8 but found 6

(indentation)